app/lang/translations.js

```diff
@@ -157,4 +157,83 @@ export default {
     "pollTypes.findADate.label": "Ich möchte einen Termin finden. (z.B. Datum oder Uhrzeit für eine Verabredung)",
     "pollTypes.makeAPoll.label": "Ich möchte eine Frage beantworten. (z.B. Wozu schmeckt Kaffe am Besten?)",
   }
```

Comment:
Thank you very much for spanish translation.
Build fails due to missing comma. Please fix the pull request and I will merge.